_data/categories.json

```diff
@@ -0,0 +1,16 @@
+[
+  "defense",
+  "InitiativesAggregation" ,
+  "volunteering" ,
+  "tech-initiatives" ,
+  "EquipmentNFood" ,
```

Comment:
Maybe a good opportunity to unify the convention here (I see at least 3)

Reply:
Unless it causes backwards compatibility issue for the existing links, in which case probably best to handle in a separate PR

Reply:
I totally agree on the need (there are more issues, for example using "Hebrish" like "Needarim"
However I wanted to keep this PR as format change only, opened: https://github.com/4tals/LinksForIsrael/issues/227